frontend/amundsen_application/static/js/features/NavBar/index.tsx

```diff
@@ -1,146 +1,211 @@
+  }
 
-    return (
-      <nav className="container-fluid">
-        <div className="row">
-          <div className="nav-bar">
-            <div id="nav-bar-left" className="nav-bar-left">
-              <Link to="/">
-                {AppConfig.logoPath && (
-                  <img
-                    id="logo-icon"
-                    className="logo-icon"
-                    src={AppConfig.logoPath}
-                    alt=""
-                  />
-                )}
-                <span className="title-3">{getLogoTitle()}</span>
-              </Link>
-            </div>
-            {this.renderSearchBar()}
-            <div id="nav-bar-right" className="ml-auto nav-bar-right">
-              {this.generateNavLinks(getNavLinks())}
-              {feedbackEnabled() && <Feedback />}
-              {loggedInUser && indexUsersEnabled() && (
-                <Dropdown id="user-dropdown" pullRight>
-                  <Dropdown.Toggle
-                    noCaret
-                    className="nav-bar-avatar avatar-dropdown"
-                  >
-                    {avatar}
-                  </Dropdown.Toggle>
-                  <Dropdown.Menu className="profile-menu">
-                    <div className="profile-menu-header">
-                      <div className="title-2">{loggedInUser.display_name}</div>
-                      <div>{loggedInUser.email}</div>
-                    </div>
-                    <MenuItem
-                      componentClass={Link}
-                      id="nav-bar-avatar-link"
-                      to={userLink}
-                      href={userLink}
-                    >
-                      {PROFILE_LINK_TEXT}
-                    </MenuItem>
-                  </Dropdown.Menu>
-                </Dropdown>
-              )}
-              {loggedInUser && !indexUsersEnabled() && (
-                <div className="nav-bar-avatar">{avatar}</div>
+  const handleTourClick = () => {
+    setRunTour(true);
+  };
+
+  const handleTourEnd = () => {
+    setRunTour(false);
+  };
+
+  return (
+    <nav className="container-fluid">
+      <div className="row">
+        <div className="nav-bar">
+          <div id="nav-bar-left" className="nav-bar-left">
+            <Link to="/">
+              {AppConfig.logoPath && (
+                <img
+                  id="logo-icon"
+                  className="logo-icon"
+                  src={AppConfig.logoPath}
+                  alt=""
+                />
               )}
-            </div>
+              <span className="title-3">{getLogoTitle()}</span>
+            </Link>
+          </div>
+          {renderSearchBar(location)}
+          <div id="nav-bar-right" className="ml-auto nav-bar-right">
+            {generateNavLinks(getNavLinks())}
+            {hasPageTour && <ProductTourButton onClick={handleTourClick} />}
+            {feedbackEnabled() && <Feedback />}
+            {loggedInUser && indexUsersEnabled() && (
```

Comment:
From the screenshot, it looks like the user dropdown circle icon is higher and closer to the feedback icon than the rest of the links/icons in the nav bar are in relation to each other - I see that is how it currently is too, but it seems a bit more noticeable now being next to the icons that are all aligned and equally spaced from the nav links. what do you think about lowering it and bringing it right by a few px?

Reply:
Good call!
I will review that spacing right now!